AirCasting/SessionViews/ChartViewModel.swift

```diff
@@ -13,9 +13,15 @@ final class ChartViewModel: ObservableObject {
     @Published var chartEndTime: Date?
     @ObservedObject var session: SessionEntity
 
+    private var databaseObserver: ChartDatabaseObserver?
     var stream: MeasurementStreamEntity? {
         didSet {
-            guard session.isActive || session.isFollowed || session.status == .NEW else { return }
+            databaseObserver = nil
+            guard shouldBeUpdatingChartForCurrentSession(), let sensor = stream?.sensorName else { return }
+            databaseObserver = ChartDatabaseObserver(session: session.uuid, sensor: sensor) {
+                Log.info("Measurements change detected [\(self.session.uuid ?? "??")]")
+                self.generateEntries()
+            }
             generateEntries()
```

Comment:
Do we want to use  `generateEntries()`, despite we used it above? If so, could you tell me why, please

Reply:
note that one call will be executed right away (the latter), but the other will execute only after a measurements change.